server/src/test/java/org/opensearch/repositories/RepositoriesServiceTests.java

```diff
@@ -210,15 +210,15 @@ private void assertThrowsOnRegister(String repoName) {
         expectThrows(RepositoryException.class, () -> repositoriesService.registerRepository(request, null));
     }
 
-    private static class TestRepository implements Repository {
+    public static class TestRepository implements Repository {
```

Comment:
dont make them public from one test class. This would add un-necessary dependency across test classes.

Reply:
This test class already has a few mock objects of respositoriesService which could be leveraged, else the same code had to be duplicated here as well for mocking purpose.

Reply:
This has been revoked.